website/common/script/ops/scoreTask.js

```diff
@@ -40,6 +40,13 @@ function _calculateDelta (task, direction, cron) {
     }
   }
 
+  // ===== CRITICAL HITS =====
```

Comment:
So critical hits now affect the value of tasks? I'm not sure that's something we want.

Also, what happens when a task is unchecked? Since critical hits can only happen on task completion, the task value will never go back down to what it was before.